testing/py3-compdb/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Leo <thinkabit.ukim@gmail.com>
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=py3-compdb
+_pkgname=${pkgname#py3-*}
+pkgver=0.2.0
+pkgrel=0
+pkgdesc="The compilation database Swiss army knife"
+url="https://github.com/Sarcasm/compdb"
+arch="noarch"
+license="MIT"
+depends="py3-setuptools"
```

Comment:
I bet it should be `makedepends`

Reply:
no, it has `/usr/bin/compdb` that has

```
#!/usr/bin/python3
# EASY-INSTALL-ENTRY-SCRIPT: 'compdb==0.2.0','console_scripts','compdb'
__requires__ = 'compdb==0.2.0'
import re
import sys
from pkg_resources import load_entry_point

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(
        load_entry_point('compdb==0.2.0', 'console_scripts', 'compdb')()
    )
```

so `py3-setuptools` which provides `pkg-resources` is required.